README.md

```diff
@@ -1,25 +1,236 @@
-# microverse-ltr
-## Hi 👋, I'm Safari a student at microverse
+<a name="readme-top"></a>
 
-I’m currently working on [ portfolio-project ](https://danielsafari143.github.io/)
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-# Built With
+Modify this file to match your project and remove sections that don't apply.
 
-    -HTML
-    -CSS
-    -GIITHUB
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
+After you're finished please remove all the comments and instructions!
+-->
 
-## Authors
+<div align="center">
 
-**Author1**
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [OPTIONAL ] This image is broken on your readme template. This is probably because you haven't added the [murple_logo.png](https://github.com/microverseinc/readme-template/blob/master/murple_logo.png) image to your project. Kindly add it for it to appear